### PR TITLE
[MOSIP-32045] Removing VIRUS_SCAN ,PRINT_POSTAL_SERVICE

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: db_scripts/mosip_regprc/dml/regprc-transaction_type.csv [hunk 1]
```diff
@@ -8,16 +8,13 @@ BIO,transaction_done,eng,TRUE,MOSIP_SYSTEM,now()
...
 VALIDATE_PACKET,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
-VIRUS_SCAN,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 OSI_VALIDATE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 EXTERNAL_INTEGRATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 DEMOGRAPHIC_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 BIOGRAPHIC_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 UIN_GENERATOR,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 NOTIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
-PRINT_SERVICE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
```
Comment (sowmya695) [inline db_scripts/mosip_regprc/dml/regprc-transaction_type.csv:19]: PRINT_SERVICE is currently being used,I have requested for removal of PRINT

Comment (Khuddusshariff0022) [reply db_scripts/mosip_regprc/dml/regprc-transaction_type.csv:19]: added PRINT_SERVICE

File: db_scripts/default_data_cleanup/cleanup.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DELETE FROM regprc.transaction_type where code ='VIRUS_SCAN';;
```
Comment (sowmya695) [inline db_scripts/default_data_cleanup/cleanup.sql]: Two ; please remove it

Comment (Khuddusshariff0022) [reply db_scripts/default_data_cleanup/cleanup.sql]: Done

**Review by sowmya695 — APPROVED** [review]

**Review by vishwa-vyom — APPROVED** [review]

**Review by sowmya695 — APPROVED** [review]

**Pull request merged**